diff -aur hycom/ALL/archive/src/Make_all.src hycom-Ubuntu-10.04/ALL/archive/src/Make_all.src
--- hycom/ALL/archive/src/Make_all.src  2005-03-02 12:41:09.000000000 -0500
+++ hycom-Ubuntu-10.04/ALL/archive/src/Make_all.src     2016-06-07 14:42:48.778077614 -0400
@@ -15,4 +15,4 @@
 #setenv ARCH t3e
 #setenv ARCH x1
 #
-setenv ARCH amd64
+setenv ARCH gfortran
diff -aur hycom/ALL/archive/src/Makefile hycom-Ubuntu-10.04/ALL/archive/src/Makefile
--- hycom/ALL/archive/src/Makefile      2012-11-01 16:43:31.000000000 -0400
+++ hycom-Ubuntu-10.04/ALL/archive/src/Makefile 2016-06-08 13:11:45.528125659 -0400
@@ -120,31 +120,31 @@
        $(LD) $(LDFLAGS) -o field2data    field2data.o    $(MODB) $(DATAH_OBJ)

 field2ncdf:         $(MODB) field2data.o    $(NCDFH_OBJ)
-       $(LD) $(LDFLAGS) -o field2ncdf    field2data.o    $(MODB) $(NCDFH_OBJ) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o field2ncdf    field2data.o    $(MODB) $(NCDFH_OBJ) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 archv2ncdf2d:       $(MODB) archv2data2d.o  $(NCDFH_OBJ)
-       $(LD) $(LDFLAGS) -o archv2ncdf2d  archv2data2d.o  $(MODB) $(NCDFH_OBJ) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o archv2ncdf2d  archv2data2d.o  $(MODB) $(NCDFH_OBJ) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 archm2ncdf2d:       $(MODB) archm2data2d.o  $(NCDFH_OBJ)
-       $(LD) $(LDFLAGS) -o archm2ncdf2d  archm2data2d.o  $(MODB) $(NCDFH_OBJ) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o archm2ncdf2d  archm2data2d.o  $(MODB) $(NCDFH_OBJ) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 archv2ncdf2t:       $(MODB) archv2data2t.o  $(NCDFH_OBJ)
-       $(LD) $(LDFLAGS) -o archv2ncdf2t  archv2data2t.o  $(MODB) $(NCDFH_OBJ) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o archv2ncdf2t  archv2data2t.o  $(MODB) $(NCDFH_OBJ) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 archv2ncdf3z:       $(MODZ) archv2data3z.o  $(NCDFZ_OBJ)
-       $(LD) $(LDFLAGS) -o archv2ncdf3z  archv2data3z.o  $(MODZ) $(NCDFZ_OBJ) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o archv2ncdf3z  archv2data3z.o  $(MODZ) $(NCDFZ_OBJ) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 archm2ncdf3z:       $(MODZ) archm2data3z.o  $(NCDFZ_OBJ)
-       $(LD) $(LDFLAGS) -o archm2ncdf3z  archm2data3z.o  $(MODZ) $(NCDFZ_OBJ) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o archm2ncdf3z  archm2data3z.o  $(MODZ) $(NCDFZ_OBJ) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 archv2ncdfsf:       $(MODB) archv2datasf.o  $(NCDFZ_OBJ)
-       $(LD) $(LDFLAGS) -o archv2ncdfsf  archv2datasf.o  $(MODB) $(NCDFZ_OBJ) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o archv2ncdfsf  archv2datasf.o  $(MODB) $(NCDFZ_OBJ) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 archv2ncdfsfl:      $(MODB) archv2datasfl.o $(NCDFZ_OBJ)
-       $(LD) $(LDFLAGS) -o archv2ncdfsfl archv2datasfl.o $(MODB) $(NCDFZ_OBJ) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o archv2ncdfsfl archv2datasfl.o $(MODB) $(NCDFZ_OBJ) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 archv2ncdfsfz:     $(MODB) archv2datasfz.o $(NCDFZ_OBJ)
-       $(LD) $(LDFLAGS) -o archv2ncdfsfz archv2datasfz.o $(MODB) $(NCDFZ_OBJ) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o archv2ncdfsfz archv2datasfz.o $(MODB) $(NCDFZ_OBJ) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 archv2ncombc:       $(MODB) archv2ncombc.o  $(DATAH_OBJ)
        $(LD) $(LDFLAGS) -o archv2ncombc  archv2ncombc.o  $(MODB) $(DATAH_OBJ)
diff -aur hycom/ALL/archive/src_2.2.35/Makefile hycom-Ubuntu-10.04/ALL/archive/src_2.2.35/Makefile
--- hycom/ALL/archive/src_2.2.35/Makefile       2012-11-01 16:43:31.000000000 -0400
+++ hycom-Ubuntu-10.04/ALL/archive/src_2.2.35/Makefile  2016-06-08 13:11:45.528125659 -0400
@@ -120,31 +120,31 @@
        $(LD) $(LDFLAGS) -o field2data    field2data.o    $(MODB) $(DATAH_OBJ)

 field2ncdf:         $(MODB) field2data.o    $(NCDFH_OBJ)
-       $(LD) $(LDFLAGS) -o field2ncdf    field2data.o    $(MODB) $(NCDFH_OBJ) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o field2ncdf    field2data.o    $(MODB) $(NCDFH_OBJ) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 archv2ncdf2d:       $(MODB) archv2data2d.o  $(NCDFH_OBJ)
-       $(LD) $(LDFLAGS) -o archv2ncdf2d  archv2data2d.o  $(MODB) $(NCDFH_OBJ) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o archv2ncdf2d  archv2data2d.o  $(MODB) $(NCDFH_OBJ) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 archm2ncdf2d:       $(MODB) archm2data2d.o  $(NCDFH_OBJ)
-       $(LD) $(LDFLAGS) -o archm2ncdf2d  archm2data2d.o  $(MODB) $(NCDFH_OBJ) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o archm2ncdf2d  archm2data2d.o  $(MODB) $(NCDFH_OBJ) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 archv2ncdf2t:       $(MODB) archv2data2t.o  $(NCDFH_OBJ)
-       $(LD) $(LDFLAGS) -o archv2ncdf2t  archv2data2t.o  $(MODB) $(NCDFH_OBJ) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o archv2ncdf2t  archv2data2t.o  $(MODB) $(NCDFH_OBJ) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 archv2ncdf3z:       $(MODZ) archv2data3z.o  $(NCDFZ_OBJ)
-       $(LD) $(LDFLAGS) -o archv2ncdf3z  archv2data3z.o  $(MODZ) $(NCDFZ_OBJ) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o archv2ncdf3z  archv2data3z.o  $(MODZ) $(NCDFZ_OBJ) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 archm2ncdf3z:       $(MODZ) archm2data3z.o  $(NCDFZ_OBJ)
-       $(LD) $(LDFLAGS) -o archm2ncdf3z  archm2data3z.o  $(MODZ) $(NCDFZ_OBJ) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o archm2ncdf3z  archm2data3z.o  $(MODZ) $(NCDFZ_OBJ) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 archv2ncdfsf:       $(MODB) archv2datasf.o  $(NCDFZ_OBJ)
-       $(LD) $(LDFLAGS) -o archv2ncdfsf  archv2datasf.o  $(MODB) $(NCDFZ_OBJ) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o archv2ncdfsf  archv2datasf.o  $(MODB) $(NCDFZ_OBJ) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 archv2ncdfsfl:      $(MODB) archv2datasfl.o $(NCDFZ_OBJ)
-       $(LD) $(LDFLAGS) -o archv2ncdfsfl archv2datasfl.o $(MODB) $(NCDFZ_OBJ) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o archv2ncdfsfl archv2datasfl.o $(MODB) $(NCDFZ_OBJ) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 archv2ncdfsfz:     $(MODB) archv2datasfz.o $(NCDFZ_OBJ)
-       $(LD) $(LDFLAGS) -o archv2ncdfsfz archv2datasfz.o $(MODB) $(NCDFZ_OBJ) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o archv2ncdfsfz archv2datasfz.o $(MODB) $(NCDFZ_OBJ) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 archv2ncombc:       $(MODB) archv2ncombc.o  $(DATAH_OBJ)
        $(LD) $(LDFLAGS) -o archv2ncombc  archv2ncombc.o  $(MODB) $(DATAH_OBJ)
diff -aur hycom/ALL/bin/Make_all.com hycom-Ubuntu-10.04/ALL/bin/Make_all.com
--- hycom/ALL/bin/Make_all.com  2012-08-14 13:05:13.000000000 -0400
+++ hycom-Ubuntu-10.04/ALL/bin/Make_all.com     2016-06-07 15:12:52.074123245 -0400
@@ -11,7 +11,7 @@
   endif
 # setenv OS LinuxIFC
 # setenv OS LinuxICE
-# setenv OS LinuxGF
+setenv OS LinuxGF
 # setenv OS XT5
 endif
 #if ($OS == "SunOS") then
diff -aur hycom/ALL/bin/Make_ncdf.com hycom-Ubuntu-10.04/ALL/bin/Make_ncdf.com
--- hycom/ALL/bin/Make_ncdf.com 2012-11-02 13:02:08.000000000 -0400
+++ hycom-Ubuntu-10.04/ALL/bin/Make_ncdf.com    2016-06-08 14:01:56.596201851 -0400
@@ -27,6 +27,7 @@
   endif
 # setenv OS LinuxIFC
 # setenv OS LinuxICE
+setenv OS LinuxGF
 # setenv OS XT5
 endif
 #if ($OS == "SunOS") then
@@ -43,6 +44,14 @@
        setenv CC       "gcc"
        setenv CFLAGS   "-O -march=k8 -m64 -mcmodel=medium"
        breaksw
+case 'LinuxGF':
+#       compile for gfortran
+       setenv FC       "gfortran"
+       setenv FFLAGS   "-fPIC -fno-second-underscore -fconvert=big-endian -O"
+       setenv FLIBS    ""
+       setenv CC       "gcc"
+       setenv CFLAGS   "-fPIC -fno-second-underscore -O"
+       breaksw
 case 'LinuxICE':
 #       compile for SGI Altix ICE, Intel compiler
        setenv FC       "ifort"
@@ -117,9 +126,9 @@
 #
 foreach f ( hycom_binning_nc hycom_scrip_nc )
   if ( ! -e ${f}_${OS} ) then
-    $FC $FFLAGS ${f}.F hycom_endian_io.o parse.o libnetcdf.a ${EXTRANCDF} -o ${f}_${OS}
+    $FC $FFLAGS ${f}.F hycom_endian_io.o parse.o /usr/lib/libnetcdff.a libnetcdf.a ${EXTRANCDF} -o ${f}_${OS}
   else if ( -f `find ${f}.F -prune -newer ${f}_${OS}` ) then
-    $FC $FFLAGS ${f}.F hycom_endian_io.o parse.o libnetcdf.a ${EXTRANCDF} -o ${f}_${OS}
+    $FC $FFLAGS ${f}.F hycom_endian_io.o parse.o /usr/lib/libnetcdff.a libnetcdf.a ${EXTRANCDF} -o ${f}_${OS}
   else
     echo "${f}_${OS} is already up to date"
   endif
@@ -131,9 +140,9 @@
 #
 foreach f ( hycom_profile2z_nc hycom_profile2s_nc hycom_seaice_nc )
   if ( ! -e ${f}_${OS} ) then
-    $FC $FFLAGS ${f}.F hycom_profile_lib.o hycom_endian_io.o parse.o libnetcdf.a ${EXTRANCDF} -o ${f}_${OS}
+    $FC $FFLAGS ${f}.F hycom_profile_lib.o hycom_endian_io.o parse.o /usr/lib/libnetcdff.a libnetcdf.a ${EXTRANCDF} -o ${f}_${OS}
   else if ( -f `find ${f}.F -prune -newer ${f}_${OS}` ) then
-    $FC $FFLAGS ${f}.F hycom_profile_lib.o hycom_endian_io.o parse.o libnetcdf.a ${EXTRANCDF} -o ${f}_${OS}
+    $FC $FFLAGS ${f}.F hycom_profile_lib.o hycom_endian_io.o parse.o /usr/lib/libnetcdff.a libnetcdf.a ${EXTRANCDF} -o ${f}_${OS}
   else
     echo "${f}_${OS} is already up to date"
   endif
@@ -145,13 +154,14 @@
 #
 foreach f ( wind_stat_nc wind_stat_range_nc )
   if ( ! -e ${f}_${OS} ) then
-    $FC $FFLAGS ${f}.f libnetcdf.a ${EXTRANCDF} -o ${f}_${OS}
+    $FC $FFLAGS ${f}.f /usr/lib/libnetcdff.a libnetcdf.a ${EXTRANCDF} -o ${f}_${OS}
   else if ( -f `find ${f}.f -prune -newer ${f}_${OS}` ) then
-    $FC $FFLAGS ${f}.f libnetcdf.a ${EXTRANCDF} -o ${f}_${OS}
+    $FC $FFLAGS ${f}.f /usr/lib/libnetcdff.a libnetcdf.a ${EXTRANCDF} -o ${f}_${OS}
   else
     echo "${f}_${OS} is already up to date"
   endif
   touch       ${f}
   /bin/rm -f  ${f}
   chmod a+rx  ${f}_${OS}
+  /bin/ln -s  ${f}_${OS} ${f}
 end
diff -aur hycom/ALL/force/src/Makefile hycom-Ubuntu-10.04/ALL/force/src/Makefile
--- hycom/ALL/force/src/Makefile        2012-11-01 16:44:33.000000000 -0400
+++ hycom-Ubuntu-10.04/ALL/force/src/Makefile   2016-06-08 13:31:32.328155690 -0400
@@ -135,22 +135,22 @@
        $(LD) $(LDFLAGS) conv_w_mm5.o  $(MODS)  zh.o    -o conv_w_mm5

 force2nc:                            force2nc.o $(MODS) zh.o
-       $(LD) $(LDFLAGS) -o force2nc force2nc.o $(MODS) zh.o    libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o force2nc force2nc.o $(MODS) zh.o    /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 nrl2nc:                              nrl2nc.o   $(MODS) zh.o
-       $(LD) $(LDFLAGS) -o nrl2nc   nrl2nc.o   $(MODS) zh.o    libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o nrl2nc   nrl2nc.o   $(MODS) zh.o    /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 ap_nc:                       $(MODS) ap_nc.o            $(LIBS)
-       $(LD) $(LDFLAGS) -o ap_nc    ap_nc.o    $(MODS) $(LIBS) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o ap_nc    ap_nc.o    $(MODS) $(LIBS) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 kp_nc:                       $(MODS) kp_nc.o            $(LIBS)
-       $(LD) $(LDFLAGS) -o kp_nc    kp_nc.o    $(MODS) $(LIBS) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o kp_nc    kp_nc.o    $(MODS) $(LIBS) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 tp_nc:                       $(MODS) tp_nc.o            $(LIBS)
-       $(LD) $(LDFLAGS) -o tp_nc    tp_nc.o    $(MODS) $(LIBS) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o tp_nc    tp_nc.o    $(MODS) $(LIBS) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 wi_nc:                       $(MODP) wi_nc.o            $(LIBP)
-       $(LD) $(LDFLAGS) -o wi_nc    wi_nc.o    $(MODP) $(LIBP) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o wi_nc    wi_nc.o    $(MODP) $(LIBP) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 mod_scrip.o: mod_scrip.f

diff -aur hycom/ALL/force/src_2.2.29/Makefile hycom-Ubuntu-10.04/ALL/force/src_2.2.29/Makefile
--- hycom/ALL/force/src_2.2.29/Makefile 2012-11-01 16:44:33.000000000 -0400
+++ hycom-Ubuntu-10.04/ALL/force/src_2.2.29/Makefile    2016-06-08 13:31:32.328155690 -0400
@@ -135,22 +135,22 @@
        $(LD) $(LDFLAGS) conv_w_mm5.o  $(MODS)  zh.o    -o conv_w_mm5

 force2nc:                            force2nc.o $(MODS) zh.o
-       $(LD) $(LDFLAGS) -o force2nc force2nc.o $(MODS) zh.o    libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o force2nc force2nc.o $(MODS) zh.o    /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 nrl2nc:                              nrl2nc.o   $(MODS) zh.o
-       $(LD) $(LDFLAGS) -o nrl2nc   nrl2nc.o   $(MODS) zh.o    libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o nrl2nc   nrl2nc.o   $(MODS) zh.o    /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 ap_nc:                       $(MODS) ap_nc.o            $(LIBS)
-       $(LD) $(LDFLAGS) -o ap_nc    ap_nc.o    $(MODS) $(LIBS) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o ap_nc    ap_nc.o    $(MODS) $(LIBS) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 kp_nc:                       $(MODS) kp_nc.o            $(LIBS)
-       $(LD) $(LDFLAGS) -o kp_nc    kp_nc.o    $(MODS) $(LIBS) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o kp_nc    kp_nc.o    $(MODS) $(LIBS) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 tp_nc:                       $(MODS) tp_nc.o            $(LIBS)
-       $(LD) $(LDFLAGS) -o tp_nc    tp_nc.o    $(MODS) $(LIBS) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o tp_nc    tp_nc.o    $(MODS) $(LIBS) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 wi_nc:                       $(MODP) wi_nc.o            $(LIBP)
-       $(LD) $(LDFLAGS) -o wi_nc    wi_nc.o    $(MODP) $(LIBP) libnetcdf.a $(EXTRANCDF)
+       $(LD) $(LDFLAGS) -o wi_nc    wi_nc.o    $(MODP) $(LIBP) /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 mod_scrip.o: mod_scrip.f

diff -aur hycom/ALL/Make_ncdf.src hycom-Ubuntu-10.04/ALL/Make_ncdf.src
--- hycom/ALL/Make_ncdf.src     2012-11-02 13:02:40.000000000 -0400
+++ hycom-Ubuntu-10.04/ALL/Make_ncdf.src        2016-06-08 13:15:30.124131342 -0400
@@ -12,9 +12,9 @@
 #setenv NCDF /common/netcdf-3.6.2-amd64pg
 #setenv NCDF /common/netcdf/deprecated/netcdf-3.6.2-amd64pg
 #
-setenv NCDF /common/netcdf/pgi11.8/3.6.3
+setenv NCDF /usr
 #
 # --- Use EXTRANCDF for the extra libraries needed for NCDF v4
 #
 #setenv EXTRANCDF "/u/home/wallcraf/pkgs/hdf5-1.8.4-serial/lib/libhdf5_hl.a /u/home/wallcraf/pkgs/hdf5-1.8.4-serial/lib/libhdf5.a /u/home/wallcraf/pkgs/zlib-1.2.3/lib/libz.a"
-setenv EXTRANCDF ""
+setenv EXTRANCDF "/usr/lib/libnetcdff.a /usr/lib/x86_64-linux-gnu/libhdf5_hl.a /usr/lib/x86_64-linux-gnu/libhdf5.a /usr/lib/x86_64-linux-gnu/libz.a -lpthread -lcurl -ldl "
diff -aur hycom/ALL/plot/src/Makefile hycom-Ubuntu-10.04/ALL/plot/src/Makefile
--- hycom/ALL/plot/src/Makefile 2012-11-01 16:45:46.000000000 -0400
+++ hycom-Ubuntu-10.04/ALL/plot/src/Makefile    2016-06-08 15:04:17.776296519 -0400
@@ -81,9 +81,9 @@
                tracks.f \
                mod_xc.F mod_za.F mod_za_sm.F wtime.F

-NAVO_OBJ =     $(PROC_OBJ) getdat_nc.o libnetcdf.a $(EXTRANCDF)
+NAVO_OBJ =     $(PROC_OBJ) getdat_nc.o /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

-NAVO_LIB =     libnetcdf.a $(EXTRANCDF)
+NAVO_LIB =     /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 NAVO_SRC =     $(PROC_SRC) getdat_nc.f

diff -aur hycom/ALL/plot/src_2.1.00/Makefile hycom-Ubuntu-10.04/ALL/plot/src_2.1.00/Makefile
--- hycom/ALL/plot/src_2.1.00/Makefile  2012-11-01 16:45:46.000000000 -0400
+++ hycom-Ubuntu-10.04/ALL/plot/src_2.1.00/Makefile     2016-06-08 15:04:17.776296519 -0400
@@ -81,9 +81,9 @@
                tracks.f \
                mod_xc.F mod_za.F mod_za_sm.F wtime.F

-NAVO_OBJ =     $(PROC_OBJ) getdat_nc.o libnetcdf.a $(EXTRANCDF)
+NAVO_OBJ =     $(PROC_OBJ) getdat_nc.o /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

-NAVO_LIB =     libnetcdf.a $(EXTRANCDF)
+NAVO_LIB =     /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 NAVO_SRC =     $(PROC_SRC) getdat_nc.f

diff -aur hycom/ALL/relax/src/Makefile hycom-Ubuntu-10.04/ALL/relax/src/Makefile
--- hycom/ALL/relax/src/Makefile        2012-11-02 12:24:20.000000000 -0400
+++ hycom-Ubuntu-10.04/ALL/relax/src/Makefile   2016-06-08 13:35:36.940161879 -0400
@@ -37,7 +37,7 @@

 MODS = mod_xc.o mod_za.o wtime.o
 LIBS = blkdat.o interp.o zh.o
-LIBN = blkdat.o interp.o zh.o libnetcdf.a $(EXTRANCDF)
+LIBN = blkdat.o interp.o zh.o /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 # select one of the programs to make,
 #  e.g. make z_levitus ARCH=sun >& Make_z_levitus
diff -aur hycom/ALL/relax/src_2.2.35/Makefile hycom-Ubuntu-10.04/ALL/relax/src_2.2.35/Makefile
--- hycom/ALL/relax/src_2.2.35/Makefile 2012-11-02 12:24:20.000000000 -0400
+++ hycom-Ubuntu-10.04/ALL/relax/src_2.2.35/Makefile    2016-06-08 13:35:36.940161879 -0400
@@ -37,7 +37,7 @@

 MODS = mod_xc.o mod_za.o wtime.o
 LIBS = blkdat.o interp.o zh.o
-LIBN = blkdat.o interp.o zh.o libnetcdf.a $(EXTRANCDF)
+LIBN = blkdat.o interp.o zh.o /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 # select one of the programs to make,
 #  e.g. make z_levitus ARCH=sun >& Make_z_levitus
diff -aur hycom/ALL/topo/src/Makefile hycom-Ubuntu-10.04/ALL/topo/src/Makefile
--- hycom/ALL/topo/src/Makefile 2012-11-02 12:18:33.000000000 -0400
+++ hycom-Ubuntu-10.04/ALL/topo/src/Makefile    2016-06-08 15:12:31.172309004 -0400
@@ -34,7 +34,7 @@
        /bin/rm *.o *.mod M*log *hgram *min *1d *2d *2h *ppmX *sea *oth *clip *ff *flat *fill *mask *set *shrink *sub *map *nam *sby *trop *tor *rge *fy *it *noshrink *son *ude *zcells *zthin *arctic _test *_ns *1deg  *lat *ts *ed *pe *gh *ip *es*30sec *pi *sea-b *360 *island

 MODS = mod_xc.o mod_za.o wtime.o
-MODN = mod_xc.o mod_za.o wtime.o libnetcdf.a $(EXTRANCDF)
+MODN = mod_xc.o mod_za.o wtime.o /usr/lib/libnetcdff.a /usr/lib/libnetcdf.a $(EXTRANCDF)
 MODG = $(MODS) spherdist.o rotang.o

 # update ARCH in Make_all.com and
diff -aur hycom/ALL/topo/src_2.2.72/Makefile hycom-Ubuntu-10.04/ALL/topo/src_2.2.72/Makefile
--- hycom/ALL/topo/src_2.2.72/Makefile  2012-11-02 12:18:33.000000000 -0400
+++ hycom-Ubuntu-10.04/ALL/topo/src_2.2.72/Makefile     2016-06-08 15:12:31.172309004 -0400
@@ -34,7 +34,7 @@
        /bin/rm *.o *.mod M*log *hgram *min *1d *2d *2h *ppmX *sea *oth *clip *ff *flat *fill *mask *set *shrink *sub *map *nam *sby *trop *tor *rge *fy *it *noshrink *son *ude *zcells *zthin *arctic _test *_ns *1deg  *lat *ts *ed *pe *gh *ip *es*30sec *pi *sea-b *360 *island

 MODS = mod_xc.o mod_za.o wtime.o
-MODN = mod_xc.o mod_za.o wtime.o libnetcdf.a $(EXTRANCDF)
+MODN = mod_xc.o mod_za.o wtime.o /usr/lib/libnetcdff.a /usr/lib/libnetcdf.a $(EXTRANCDF)
 MODG = $(MODS) spherdist.o rotang.o

 # update ARCH in Make_all.com and
diff -aur hycom/ATLb2.00/force/plot/src/Makefile hycom-Ubuntu-10.04/ATLb2.00/force/plot/src/Makefile
--- hycom/ATLb2.00/force/plot/src/Makefile      2012-11-01 16:45:46.000000000 -0400
+++ hycom-Ubuntu-10.04/ATLb2.00/force/plot/src/Makefile 2016-06-08 15:04:17.776296519 -0400
@@ -81,9 +81,9 @@
                tracks.f \
                mod_xc.F mod_za.F mod_za_sm.F wtime.F

-NAVO_OBJ =     $(PROC_OBJ) getdat_nc.o libnetcdf.a $(EXTRANCDF)
+NAVO_OBJ =     $(PROC_OBJ) getdat_nc.o /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

-NAVO_LIB =     libnetcdf.a $(EXTRANCDF)
+NAVO_LIB =     /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 NAVO_SRC =     $(PROC_SRC) getdat_nc.f

diff -aur hycom/ATLb2.00/plot/src/Makefile hycom-Ubuntu-10.04/ATLb2.00/plot/src/Makefile
--- hycom/ATLb2.00/plot/src/Makefile    2012-11-01 16:45:46.000000000 -0400
+++ hycom-Ubuntu-10.04/ATLb2.00/plot/src/Makefile       2016-06-08 15:04:17.776296519 -0400
@@ -81,9 +81,9 @@
                tracks.f \
                mod_xc.F mod_za.F mod_za_sm.F wtime.F

-NAVO_OBJ =     $(PROC_OBJ) getdat_nc.o libnetcdf.a $(EXTRANCDF)
+NAVO_OBJ =     $(PROC_OBJ) getdat_nc.o /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

-NAVO_LIB =     libnetcdf.a $(EXTRANCDF)
+NAVO_LIB =     /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 NAVO_SRC =     $(PROC_SRC) getdat_nc.f

diff -aur hycom/ATLb2.00/relax/plot/src/Makefile hycom-Ubuntu-10.04/ATLb2.00/relax/plot/src/Makefile
--- hycom/ATLb2.00/relax/plot/src/Makefile      2012-11-01 16:45:46.000000000 -0400
+++ hycom-Ubuntu-10.04/ATLb2.00/relax/plot/src/Makefile 2016-06-08 15:04:17.776296519 -0400
@@ -81,9 +81,9 @@
                tracks.f \
                mod_xc.F mod_za.F mod_za_sm.F wtime.F

-NAVO_OBJ =     $(PROC_OBJ) getdat_nc.o libnetcdf.a $(EXTRANCDF)
+NAVO_OBJ =     $(PROC_OBJ) getdat_nc.o /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

-NAVO_LIB =     libnetcdf.a $(EXTRANCDF)
+NAVO_LIB =     /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 NAVO_SRC =     $(PROC_SRC) getdat_nc.f

diff -aur hycom/GLBb1.50/plot_06/src/Makefile hycom-Ubuntu-10.04/GLBb1.50/plot_06/src/Makefile
--- hycom/GLBb1.50/plot_06/src/Makefile 2012-11-01 16:45:46.000000000 -0400
+++ hycom-Ubuntu-10.04/GLBb1.50/plot_06/src/Makefile    2016-06-08 15:04:17.776296519 -0400
@@ -81,9 +81,9 @@
                tracks.f \
                mod_xc.F mod_za.F mod_za_sm.F wtime.F

-NAVO_OBJ =     $(PROC_OBJ) getdat_nc.o libnetcdf.a $(EXTRANCDF)
+NAVO_OBJ =     $(PROC_OBJ) getdat_nc.o /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

-NAVO_LIB =     libnetcdf.a $(EXTRANCDF)
+NAVO_LIB =     /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 NAVO_SRC =     $(PROC_SRC) getdat_nc.f

diff -aur hycom/GLBb1.50/plot_07/src/Makefile hycom-Ubuntu-10.04/GLBb1.50/plot_07/src/Makefile
--- hycom/GLBb1.50/plot_07/src/Makefile 2012-11-01 16:45:46.000000000 -0400
+++ hycom-Ubuntu-10.04/GLBb1.50/plot_07/src/Makefile    2016-06-08 15:04:17.776296519 -0400
@@ -81,9 +81,9 @@
                tracks.f \
                mod_xc.F mod_za.F mod_za_sm.F wtime.F

-NAVO_OBJ =     $(PROC_OBJ) getdat_nc.o libnetcdf.a $(EXTRANCDF)
+NAVO_OBJ =     $(PROC_OBJ) getdat_nc.o /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

-NAVO_LIB =     libnetcdf.a $(EXTRANCDF)
+NAVO_LIB =     /usr/lib/libnetcdff.a libnetcdf.a $(EXTRANCDF)

 NAVO_SRC =     $(PROC_SRC) getdat_nc.f
